lib/requests/diaryFoods.js

```diff
@@ -1,15 +1,23 @@
+var foodsRequests = require('./foods')
+
 const baseURL = `https://ivmh-quantified-self-express.herokuapp.com`
 
+const mealsAPIFetch = (id, method, extension) => {
```

Comment:
I'm really intrigued by this extension idea! I wasn't sure how to handle refactoring it. Creative!